src/firefly/js/charts/ui/ColSelectView.jsx

```diff
@@ -90,9 +91,9 @@ export function hideColSelectPopup() {
 
 function popupForm(tableModel, onColSelected,buttonText,popupId, minWidth) {
     const tblId = tableModel.tbl_id;
-    popupPanelResizableStyle.minWidth = Math.min(minWidth, 560);
+    const style= clone(popupPanelResizableStyle, {minWidth: Math.min(minWidth, 560)});
```

Comment:
Frozen style objects could have interesting side effects. We did not see a warning in React 15, because it's always one component at a time that was using it.